### PR TITLE
CORE-963: Move the Blockset 'token' out of source control.

### PR DESCRIPTION
**Pull request by EBGToo**

### CODE REVIEW

**Comment by EBGToo** [comment]
Updated the README.md, as a start.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,202 @@
...
+
+To specify a `WalletKitTestsConfig.json` file do the following:
+
+* Copy the file `.../WalletKit/templates/WalletKitTestsConfig_example.json` into 
+a location in your file system, generally outside of the WalletKit cloned directory.  For example,
+`${HOME}/.walletKitTestsConfig.json` is a common location.  If you edit the file and add 
+your own paperKeys then you will want to ensure that those changes are kept independent of 
+the WalletKit directory.
+
+* Copy the file `.../WalletKit/templates/WalletKitTestsConfig.env` to
```
Comment (ewerx) [inline README.md:158]: is this template / copy step needed? could just include the template in the expected location and instruct the user to edit it if needed.

**Review by ewerx — COMMENTED** [review]
I tested instructions on a clean repo and all worked well.

File: README.md (hunk 1 above)
Comment (EBGToo) [reply README.md:158]: I didn't want a change to the .env file to appear as Git source code changes.  So `templates/` is under source control but 
```
*TestsConfig.json
/WalletKitTestsConfig.env
```
are not. (See top-level .gitignore.)

**Review by ewerx — APPROVED** [review]

**Pull request merged**